app/scripts/styles/theme.ts

```diff
@@ -79,6 +80,7 @@ export const GlobalStyles = createGlobalStyle`
 
   :root {
     --base-space-multiplier: ${themeVal('layout.glspMultiplier.xsmall')};
+    --base-font-family: ${themeVal('type.base.fontFamily')};
```

Comment:
Is there any way to set this up through uswds scss configuration?

Reply:
Thanks, I moved it to the USWDS config. We should however think about if (and how) to give the veda-ui lib users ability to override the base fonts which is the reason why I added it as a root variable initially. We don't have to do it here though as it's out of scope.

Reply:
This is such a good point. And I think this might be something that we need to resolve before merge - because each instance has its font configuration - 
The typeface is Open Sans by default, and GHG is overwriting it as Inter - https://github.com/US-GHG-Center/veda-config-ghg/blob/develop/veda.config.js#L61

Moreover, when I check the style, the calendar is rendered with `Source Sans pro` - which makes me believe that we need to handle the type overwriting of the component.

Reply:
Okay, I think I came up with a working solution and pushed the change. Thanks for having another look 🙇 

As we discussed yesterday, we will override the default typeface used by USWDS with the root base font used in our instances.

Test PR to showcase that USWDS will use the base font root variable set by the theme on the SiteAlert: https://deploy-preview-1051--veda-ui.netlify.app/

<img width="619" alt="Screenshot 2024-07-17 at 14 29 28" src="https://github.com/user-attachments/assets/a1605a33-b9e8-4e42-beed-7a63e944c838">

Also going forward I think we should import all the packages we would like to use (e.g "usa-alert", "usa-button") in one SCSS entry file (in our case the `styles.scss`) [like this](https://github.com/NASA-IMPACT/veda-ui/pull/1051/files#diff-8890a38014798c861bfe6ddab91bf46ec3c56d330831731a47453f7a1e7bce72), and only use our custom scss files per component if we would like to override some specific uswds component stylings. Otherwise, the overriding of the typeface won't work. I did a short test and this doesn't seem to affect the Parcel build time nor the CSS size.

Reply:
🥳 Wooh amazing, thanks for making it work @dzole0311 
I agree that the packages need to be in `styles.scss`. What do you think about mixins and utils? Would it be better to have a centralized point to expose those? or is it better to import them on the component levels when needed.